leader-election-impl/src/main/java/com/palantir/leader/PaxosLeaderElectionService.java

```diff
@@ -175,6 +204,10 @@ public LeadershipToken blockOnBecomingLeader() throws InterruptedException {
                     return currentState.confirmedToken().get();
                 case NO_QUORUM:
                     // If we don't have quorum we should just retry our calls.
+                    long backoffTime = ThreadLocalRandom.current().nextLong(0, noQuorumMaxDelay + 1);
```

Comment:
nit: This is just `nextLong(noQuorumMaxDelay + 1);`. Saves a bunch of tests, though to be fair this might get optimised out.